pcap-rewrite/src/filters/ipaddr_pair.rs

```diff
@@ -0,0 +1,20 @@
+use std::net::IpAddr;
+
+pub struct IpAddrPair {
+    ipaddr_0: IpAddr,
+    ipaddr_1: IpAddr,
+}
```

Comment:
This should be an anonymous struct:
```rust
pub struct IpAddrPair(IpAddr, IpAddr)
```

and you can access the fields using `.0` and `.1`

Reply:
It is even possible to make the fields visible (and remove the need for accessors like `get_ipaddr_0`):
```rust
pub struct IpAddrPair(pub IpAddr, pub IpAddr)
```